src/docfx/config/SecretConfig.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal class SecretConfig
```

Comment:
Add a comment to say all secrets MUST go here otherwise they are print out to logs